### PR TITLE
Add sig-docs-bn members 

### PR DESCRIPTION
**Pull request by mitul3737**
Adding sig docs Bengali Localization owners and reviewers.

### CODE REVIEW

File: config/kubernetes/sig-docs/teams.yaml [hunk 1]
```diff
@@ -300,6 +300,20 @@ teams:
...
     - Potapy4
     privacy: closed
+  sig-docs-bn-owners:
+    description: Admins for Bengali content
+    members:
+    - mitul3737
+    - Mouly22
+    - ikramulkayes
+    privacy: closed
+  sig-docs-es-reviews:
```
Comment (palnabarun) [inline config/kubernetes/sig-docs/teams.yaml:310]: possible nit?
```suggestion
  sig-docs-bn-reviews:
```

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: config/kubernetes/sig-docs/teams.yaml (hunk 1 above)
Comment (mitul3737) [reply config/kubernetes/sig-docs/teams.yaml:310]: Yes, need to change.

Comment (mitul3737) [reply config/kubernetes/sig-docs/teams.yaml]: p1

Can you have a look now @palnabarun ?

Comment (palnabarun) [reply config/kubernetes/sig-docs/teams.yaml]: Thanks! :)

**Comment by divya-mohan0209** [comment]
@mitul3737 : Please sign the CLA :)

**Comment by mitul3737** [comment]
@divya-mohan0209 is it done now?

**Review by divya-mohan0209 — COMMENTED** [review]
Following the existing format, could we please have this ordered alphabetically? To clarify, please move this before sig-docs-de-owners.

**Comment by mitul3737** [comment]
@divya-mohan0209 can you have a look, please? 😊

File: config/kubernetes/sig-docs/teams.yaml [hunk 2]
```diff
@@ -1,4 +1,18 @@
 teams:
+  sig-docs-bn-owners:
+    description: Admins for Bengali content
```
Comment (divya-mohan0209) [inline config/kubernetes/sig-docs/teams.yaml:3]: In line with the rest of the document, this should read as

```suggestion
    description: Approvers for Bengali content
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Nit detected in phrasing

File: config/kubernetes/sig-docs/teams.yaml (hunk 2 above)
Comment (mitul3737) [reply config/kubernetes/sig-docs/teams.yaml:3]: Okay sure!

**Comment by mitul3737** [comment]
@divya-mohan0209 Done 😊

**Comment by mitul3737** [comment]
@divya-mohan0209 anything I need to do now? As the test failed?

**Comment by divya-mohan0209** [comment]
None of you are Kubernetes org members, that's why it has failed.  You will have to request for org membership  with the help of two sponsors.

**Comment by mitul3737** [comment]
@divya-mohan0209 yes, I am not an org member but I am adding these to contribute to the Bengali documentation page. What will be the next step? What do you mean by the sponsor?

**Comment by palnabarun** [comment]
@mitul3737 -- you need to be an org member to be added as an owner/approver/reviewer to any part of the codebase. As @divya-mohan0209 said, you need to request a membership first. Also, in order to contribute to the Bengali documentation page, you don't need to be an owner/approver.

Please read the guidelines [here](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements). They will answer your questions.

**Comment by mitul3737** [comment]
@palnabarun sure!

**Comment by divya-mohan0209** [comment]
@mitul3737 : Would also request that you/one of the representatives mentioned above join the localization meeting today.  This would help further clarify any doubts you have surrounding starting up a localization.

**Comment by sftim** [comment]
/ok-to-test

(will fail, at the moment)

**Pull request closed without merging**